src/components/EditorWidgets/Markdown/MarkdownPreview/index.js

```diff
@@ -15,4 +15,8 @@ MarkdownPreview.propTypes = {
   value: PropTypes.string,
 };
 
+MarkdownPreview.defaultProps = {
+  value: '',
```

Comment:
Is there a reason we need to set this on the MD preview, and not any of the other previews?

Reply:
Are you referring to the 'RawEditor' and 'VisualEditor' files?

Reply:
As far as the kitchen sink goes, I'll take a look this evening or tomorrow afternoon.  My apologies, I did not test creating a new sink.

Reply:
No, I just noticed that you set a default value on the MarkdownPreview as well as the MarkdownControl.